rm2/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "rm2",
+  "version": "1.0.0",
+  "description": "building react app with webpack from scratch",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "webpack-dev-server --mode development --open --hot",
+    "build": "webpack --mode production"
```

Comment:
Consider adding ESLint and Prettier configs to the project.
Avoid writing your own configs, opt into community ready made plugins following e.g. AirBnB
In the "scripts" section we will need 6 new commands:
1. "eslint" and "prettier" commands to just check our code against the corresponding configs
2. "eslint:fix" and "prettier:fix" commands to check and auto fix our code against the corresponding configs
3. "lint" and "lint:fix" commands to just check both ESLint & Prettier and check & auto fix both ESLint & Prettier respectively